environments/.caas/ui-meta/slurm-infra-manila-home.yml

```diff
@@ -0,0 +1,127 @@
+# Exactly as for slurm-infra.yml but to allow for separate manila/non-manila home appliances
+name: "slurm-manila-home"
+label: "Slurm (CephFS home)"
+description: >- 
+  Batch cluster running the Slurm workload manager, the Open 
+  OnDemand web interface, and custom monitoring.
+
+  This version uses CephFS for home directories, deleted
+  when the platform is deleted.
+logo: https://upload.wikimedia.org/wikipedia/commons/thumb/3/3a/Slurm_logo.svg/158px-Slurm_logo.svg.png
+
+parameters:
+  - name: cluster_floating_ip
+    label: External IP
+    description: The external IP to use for the login node.
+    kind: cloud.ip
+    immutable: true
+
+  - name: login_flavor
+    label: Login node size
+    description: The size to use for the login node.
+    kind: cloud.size
+    immutable: true
+    options:
+      min_ram: 2048
+      min_disk: 20
+
+  - name: control_flavor
+    label: Control node size
+    description: The size to use for the control node.
+    kind: cloud.size
+    immutable: true
+    options:
+      min_ram: 2048
+      min_disk: 20
+
+  - name: compute_count
+    label: Compute node count
+    description: The number of compute nodes in the cluster.
+    kind: integer
+    options:
+      min: 1
+    default: 3
+
+  - name: compute_flavor
+    label: Compute node size
+    description: The size to use for the compute node.
+    kind: cloud.size
+    immutable: true
+    options:
+      count_parameter: compute_count
+      min_ram: 2048
+      min_disk: 20
+  
+  - name: home_volume_size
+    label: Home volume size (GB)
+    description: The size of the cloud volume or share to use for home directories.
+    kind: cloud.volume_size
```

Comment:
This is now always a share when we use this ui-meta. We should update the description and make this `integer` so it doesn't contribute to quota calculations.